### PR TITLE
+dagger

### PR DESCRIPTION
**Pull request by johnallen3d**

### CODE REVIEW

File: projects/dagger.io/package.yml [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+    mkdir -p "{{ prefix }}"/bin
+    mv dagger "{{ prefix }}"/bin
+  dependencies:
+    go.dev: ^1.18
+  env:
+    LDFLAGS:
+      - -s
+      - -w
+      - -X go.dagger.io/dagger/version.Version={{ version }}
+      - -X go.dagger.io/dagger/version.Revision=tea
```
Comment (johnallen3d) [inline projects/dagger.io/package.yml:25]: I tried to follow the Homebrew formula here but the version isn't printing as expected.

```sh
❯ /opt/homebrew/bin/dagger version
dagger 0.3.7 (2bd84ee24) darwin/arm64
❯ ~/.tea/dagger.io/v0.3.9/bin/dagger version
dagger devel () darwin/arm64
```

Comment (johnallen3d) [reply projects/dagger.io/package.yml:25]: Actually, I was looking at the forumla from `homebrew-core` but the latest is actually from [their tap](https://github.com/dagger/homebrew-tap/blob/main/dagger.rb). It looks like in this case they're pulling pre-built binaries from GitHub.

File: projects/dagger.io/package.yml [hunk 2]
```diff
@@ -11,13 +11,13 @@ provides:
...
 build:
   script: |
-    export CGO_ENABLED=0
     go build -v -ldflags="$LDFLAGS" ./cmd/dagger
     mkdir -p "{{ prefix }}"/bin
     mv dagger "{{ prefix }}"/bin
   dependencies:
     go.dev: ^1.18
   env:
+    CGO_ENABLED: 0
```
Comment (johnallen3d) [inline projects/dagger.io/package.yml:20]: TIL! 👏

**Comment by jhheider** [comment]
another one for the books!

**Comment by johnallen3d** [comment]
Thanks for your help on this one!

**Pull request merged**